compiler/testData/codegen/box/annotations/annotatedLambda/suspendLambda.kt

```diff
@@ -0,0 +1,29 @@
+// TARGET_BACKEND: JVM
+// WITH_STDLIB
+// Only implemented in the IR backend.
+// IGNORE_BACKEND: JVM
+
+import java.lang.reflect.Method
+import kotlin.test.assertEquals
+
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class Ann(val x: String)
+
+fun foo0(block: suspend () -> Unit) = block.javaClass
+
+fun testMethod(method: Method, name: String) {
+    assertEquals("OK", method.getAnnotation(Ann::class.java)?.x, "On method `${method.name}` of test named `$name`")
+}
+
+fun testClass(clazz: Class<*>, name: String) {
+    // Check that non-bridge `invoke` and `invokeSuspend` contains the suspend lambda annotation.
```

Comment:
Let's check that `invokeSuspend` contains the annotation, and `invoke` doesn't?

Reply:
Done.